ui-tests/src/main/java/org/openmrs/reference/page/FindPatientPage.java

```diff
@@ -13,7 +13,7 @@
 
     public static final By PATIENT_SEARCH_RESULT = By.id("patient-search-results-table");
```

Comment:
Let's make all those locators private instead of public so that only this page knows how it looks like.

I'll add a note the the guide.